Dockerfile

```diff
@@ -1,3 +1,33 @@
+
+###################
+FROM centos:8 as config-editor
+
+WORKDIR /config-editor
+
+RUN INSTALL_PKGS="\
+        git \
+        " && \
+    yum -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False install $INSTALL_PKGS
+
+RUN git clone https://github.com/thomasmckay/config-tool.git --branch npm /config-editor && \
```

Comment:
@alecmerdler @jonathankingfc I think this is what was mentioned in today's meeting? So yes, downstream this source would need to be checked out and provided as a build artifact. (It's pointing to my local branch due to a few minor changes.)
If there's a better way, let me know.